### DISCUSSION TITLE
Welcome to netsuite-rest Discussions!

### DISCUSSION THREAD

**Discussion by ehmad11** [ehmad11/netsuite-rest#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by charles20hk** [comment]
Hi @ehmad11 , thanks for your invitation. I've been working with Node.js for a couple of years and being a full-stack developer for many years. Recently I've involved in a project that requires the integration with Netsuite. I would like to join this community and starting with some questions or trouble would like to tackle in a short future and then may be code change in long.

**Reply by ehmad11** [reply]
hi @charles20hk, you are welcome, also your gzipped issue really helped me too, feel free to ask if you have any questions

**Comment by ghost** [comment]
Hi @ehmad11 i'm new to the development and i started as a node js developer a few months back, now i need to make a crud app with node js which can perform crud on the netsuite. Can you help me doing this?
